krun/vm_defs.py

```diff
@@ -70,6 +70,16 @@ def __init__(self, iterations_runner):
         # environment changes specific to a benchmark.
         self.common_env_changes = []
 
+        # The user can pass in a dict to influence the environment.
+        # These variables are *appended* (they do not override Krun).
```

Comment:
I see the confusion: what this comment means is that if someone specifies (e.g.) `LD_LIBRARY_PATH` then the users values are appended to those set by krun. Let's reword the comment a little bit to make that clearer.